src/main/g8/build.sbt

```diff
@@ -1,18 +1,16 @@
-lazy val akkaHttpVersion = "$akka_http_version$"
-lazy val akkaVersion    = "$akka_version$"
+lazy val akkaHttpVersion = "10.0.9"
+lazy val akkaVersion    = "2.5.3"
```

Comment:
Yes, the only reason for not using it is that it messes up the docs (which pulls in the variable names instead of the underlying values).